apps/domain/src/main/core/infrastructure/tf.py

```diff
@@ -24,7 +23,7 @@ def write(self, tfscript):
 
     def init(self):
         return subprocess.run(
-            f"terraform init -input=false -plugin-dir={ROOT_DIR}",
+            f"terraform init",
```

Comment:
We can modify this in a way such that it download plugins when plugins aren't already present. Plugins are required for faster deployment of `workers`.

Reply:
Done.